src/pages/non-pass-finalisation/non-pass-finalisation.actions.ts

```diff
@@ -1,10 +1,17 @@
 import { Action } from '@ngrx/store';
 
 export const NON_PASS_FINALISATION_VIEW_DID_ENTER = '[NonPassFinalisationPage] NonPassFinalisation view did enter';
+export const VALIDATION_ERROR = '[NonPassFinalisationPage] Validation Error';
 
 export class NonPassFinalisationViewDidEnter implements Action {
   readonly type = NON_PASS_FINALISATION_VIEW_DID_ENTER;
 }
 
+export class ValidationError implements Action {
```

Comment:
could this be prefixed with the page name so there's no confusion this one is for the Non Pass page

Reply:
Okay, sure. Was thinking not to do this so that we don't have crazy long action names. But probably you're right, should be more specific